### PR TITLE
chore: enhancement of requests

### LINKED ISSUE #16: improve /richiedi command, make it user-friendly and robust
it should be robust with the syntax (don't be so sensitive with spaces)

### PR DESCRIPTION
**Pull request by salvo-polizzi**
As recommended in #16, it was improved request command making the syntax less sensitive to whitespaces

Closes #16

### CODE REVIEW

File: module/delete.py [hunk 1]
```diff
@@ -13,6 +13,6 @@ def delete(update: Update, context: CallbackContext) -> None:
 
     rows = get_user_books(context, chat_id)
     if rows:
-        keyboard = [[InlineKeyboardButton(str(i + 1), callback_data = (DELETE_APPROVED + str(rows[i][0])))] for i in range(len(rows))]
+        keyboard = [[InlineKeyboardButton(str(i + 1), callback_data = DELETE_APPROVED + str(rows[i][0]))] for i in range(len(rows))]
```
Comment (Helias) [inline module/delete.py]: this is a bit confusing, if pylint does not complain, write it in a better way
```python
keyboard = [
    [
        InlineKeyboardButton(
            str(i + 1),
            callback_data = DELETE_APPROVED + str(rows[i][0])
        )
    ] for i in range(len(rows))
]
```

File: module/request.py [hunk 2]
```diff
@@ -31,14 +31,14 @@ def request(update: Update, context: CallbackContext) -> None:
...
+    user_isbn = message.split(';')[0].split()[1].replace(' ','')
     if len(user_isbn) != 10 and len(user_isbn) != 13:
         context.bot.send_message(chat_id, ISBN_ERROR)
         return
 
     try:
-        format(float(message.split('; ')[1].replace(",", ".")), ".2f")
-        price = str(format(float(message.split('; ')[1].replace(",", ".")), ".2f"))
+        format(float(message.split(';')[1].replace(",", ".").replace(" ","")), ".2f")
+        price = str(format(float(message.split(';')[1].replace(",", ".").replace(" ","")), ".2f"))
```
Comment (Helias) [inline module/request.py]:
```suggestion
        full_price = float(message.split(';')[1].replace(",", ".").replace(" ",""))
        price = str(format(full_price, ".2f"))
```

File: module/request.py [hunk 3]
```diff
@@ -31,21 +31,21 @@ def request(update: Update, context: CallbackContext) -> None:
...
-    user_isbn = message.split('; ')[0].split()[1]
+    user_isbn = message.split(';')[0].split()[1].replace(' ','')
     if len(user_isbn) != 10 and len(user_isbn) != 13:
         context.bot.send_message(chat_id, ISBN_ERROR)
         return
 
     try:
-        format(float(message.split('; ')[1].replace(",", ".")), ".2f")
-        price = str(format(float(message.split('; ')[1].replace(",", ".")), ".2f"))
+        format(float(message.split(';')[1].replace(",", ".").replace(" ","")), ".2f")
```
Comment (Helias) [inline module/request.py]: is this line needed?
I was splitting the `price` value into multiple variable to improve the readability

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
besides the full_price/price stuff that are minor things, I think we can merge this

File: module/request.py (hunk 3 above)
Comment (salvo-polizzi) [reply module/request.py]: No, it wasn't. My mistake. I am going to correct this.

**Pull request merged**